bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/widgets/Display.java

```diff
@@ -6832,4 +6832,30 @@ static long windowProc(long id, long sel, long arg0, long arg1, long arg2, long
 static boolean isActivateShellOnForceFocus() {
 	return "true".equals(System.getProperty("org.eclipse.swt.internal.activateShellOnForceFocus", "true")); //$NON-NLS-1$
 }
+
+/**
+ * {@return whether rescaling of shells at runtime when the DPI scaling of a
+ * shell's monitor changes is activated for this device}
```

Comment:
Yes, makes sense. I thought we might leave that out because the method always correctly returns "false", but I don't see why we should not add an according hint here as well. I've added it with 784ffc7.